app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] As the previous reviewer suggested that comments controller doesn't have a **new** action similar to the **create** action. cry Please make sure that the comments controller has new and create actions similar to the post controller in order to make your app work as expected :+1: